static/app/components/events/featureFlags/onboardingIntegrationSection.tsx

```diff
@@ -0,0 +1,133 @@
+import {Fragment, useState} from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import {Button} from 'sentry/components/button';
+import Input from 'sentry/components/input';
+import TextCopyInput from 'sentry/components/textCopyInput';
+import {IconCheckmark} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+function _usePostSecret({
+  orgSlug,
+  provider,
+}: {
+  orgSlug: string | undefined;
+  provider: string;
+}) {
+  const api = useApi();
+
+  const postSecret = async (secret: string) =>
+    await api.requestPromise(
+      `/organizations/${orgSlug}/flags/hooks/provider/${provider.toLowerCase()}/signing-secret/`,
+      {
+        method: 'POST',
+        data: {
+          secret: secret,
+        },
+      }
+    );
+
+  return {postSecret};
+}
+
+export default function OnboardingIntegrationSection({
+  provider,
+  integration,
+}: {
+  integration: string;
+  provider: string;
+}) {
+  const organization = useOrganization();
+  const [tokenSaved, setTokenSaved] = useState(false);
+  // const {postSecret} = usePostSecret({provider, orgSlug: organization?.slug});
+  const [secret, setSecret] = useState('');
+  const [storedProvider, setStoredProvider] = useState(provider);
+  const [storedIntegration, setStoredIntegration] = useState(integration);
+
+  if (provider !== storedProvider || integration !== storedIntegration) {
+    setStoredProvider(provider);
+    setStoredIntegration(integration);
+    setSecret('');
+    setTokenSaved(false);
+  }
+
+  return (
+    <Fragment>
+      <h4 style={{marginTop: space(4)}}>{t('Integrate Feature Flag Service')}</h4>
```

Comment:
Are we also styling this via `Wrapper` inside of static/app/components/events/featureFlags/featureFlagOnboardingLayout.tsx ?

Reply:
hmmm, only `margin-bottom` there, but i can move the style there